gui/src/3D/controls/touchHolographicButton.ts

```diff
@@ -234,10 +234,10 @@ export class TouchHolographicButton extends TouchButton3D {
     private _rebuildContent(): void {
         this._disposeFacadeTexture();
 
-        if (DomManagement.IsDocumentAvailable() && !!document.createElement) {
-            let panel = new StackPanel();
-            panel.isVertical = true;
+        let panel = new StackPanel();
+        panel.isVertical = true;
 
+        if (DomManagement.IsDocumentAvailable() && !!document.createElement) {

```

Comment:
Can you remove the check around the text portion of _rebuildContent in HolographicButton as well?

Since it looks like Image still needs the check, we can ignore HolographicSlate for now.

Reply:
this should now be fixed